src/helpers/storage.js

```diff
@@ -139,6 +139,18 @@ export const getParentPartitions = (deviceData, device) => {
     return deviceData[device].parents.v.map(parent => getParentPartitions(deviceData, parent)).flat(Infinity);
 };
 
+/* Get the parent disk ID of a given device
+ * @param {Object} deviceData - The device data object
+ * @param {string} device - The ID of the device
+ * @returns {string}
+ * */
+export const getDeviceDisk = (deviceData, device) => {
```

Comment:
[Feel free to ignore] I wonder if we should use the helper here: https://github.com/rhinstaller/anaconda-webui/blob/06271e44e0fda0cfdaa2c1c6be36f75eff259611/src/components/storage/MountPointMapping.jsx#L274. If so I guess it is out of scope of this PR anyway.

Reply:
Let's fix that for rawhide